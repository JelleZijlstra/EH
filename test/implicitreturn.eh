#!/usr/bin/ehi
f = _ => 42
printvar(f ())
