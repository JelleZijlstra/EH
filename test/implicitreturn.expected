%%stdout%%
@int 42
%%stderr%%
